backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -604,6 +604,14 @@ public UserInfo setCurrentUserTenantDataAccess(
     return new UserInfo(apiUser, orgRoles, isAdmin);
   }
 
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public List<ApiUser> getAllUsersByOrganization(Organization organization) {
+    Set<String> usernames = _oktaRepo.getAllUsersForOrganization(organization);
+    return usernames.stream()
+        .map(username -> _apiUserRepo.findByLoginEmailIncludeArchived(username).orElse(null))
```

Comment:
```suggestion
        .map(username -> _apiUserRepo.findByLoginEmailIncludeArchived(username).orElse(null))
        .filter(Objects::nonNull)
```
Since this can technically (although unlikely) return null, you should probably filter out those null values otherwise you could get a null pointer in the call class.

Reply:
Got it! Thank you! Will do!!